iwf.yaml

```diff
@@ -833,6 +833,8 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/TimerResult'
+        stateStartSucceeded:
```

Comment:
nit: how about `stateStartApiSucceeded` to be more clear. `stateStartSucceeded` makes it feel like the state is never being executed at all.